src/tests/suite/daos_rebuild_simple.c

```diff
@@ -29,6 +29,10 @@
 
 #define DATA_SIZE	(1048576 * 2 + 512)
 
+#if 0
+/* Disable inflight IO due to DAOS-8775 for 2.0, and re-enable it until inflight I/O
```

Comment:
should we use skip() instead?

Reply:
oh, this is general function,  and it actually does not skip any tests, just disable the inflight I/O during reintegration for each sub test.